webui_runner/Dockerfile

```diff
@@ -8,29 +9,26 @@ RUN apt-get update \
 
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -
 
-RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main"
+RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
 
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
-	clang-8 lldb-8 lld-8
+	clang-10 lldb-10 lld-10
```

Comment:
I think we should just merge this PR but if you're in this file again you could add `ENV CLANG_VERSION 10`